src/services/SQSService.js

```diff
@@ -41,7 +41,7 @@ class SQSService {
         .exec();
       const params = {
         MessageBody: {
-          subject: '',
+          subject: template.subject,
```

Comment:
The subject will be templated as well, so we need to run it through Mustache.render(...)

Reply:
done